test_conformance/images/kernel_read_write/test_common.cpp

```diff
@@ -17,3 +17,37 @@ cl_sampler create_sampler(cl_context context, image_sampler_data *sdata, bool te
     return sampler;
 }
 
+void filter_undefined_bits(image_descriptor *imageInfo, char *resultPtr)
+{
+    // mask off the top bit (bit 15) if the image format is (CL_UNORM_SHORT_555,
+    // CL_RGB). (Note: OpenCL says: the top bit is undefined meaning it can be
+    // either 0 or 1.)
+    if (imageInfo->format->image_channel_data_type == CL_UNORM_SHORT_555)
+    {
+        cl_ushort *temp = (cl_ushort *)resultPtr;
+        temp[0] &= 0xefff;
```

Comment:
0xefff is masking out bit 12, not bit 15.  Do you mean 0x7fff here?

Reply:
yeap !

Reply:
Should be fixed now (apologies for the force push on the branch(I had to rework the original commit message)

Reply:
Looks good. (Apologies for only looking at this post-merge!)